.github/workflows/ci.yml

```diff
@@ -78,10 +78,12 @@ jobs:
           flags: >-
             CI-GHA,
             pytest,
-            OS-${{
-              runner.os
-            }}
+            awx-unit,
+            OS-${{ runner.os }}
+          name: ${{ github.job }}-${{ matrix.os || runner.os }}  # Unique name per job
           token: ${{ secrets.CODECOV_TOKEN }}
+          parallel: true  # <--- ENABLE PARALLEL MODE
```

Comment:
This is a no-op.

There's no such input in this action: https://github.com/codecov/codecov-action/blob/main/action.yml

```suggestion
```